### DISCUSSION TITLE
IBasicKubernetes   file not found

### DISCUSSION THREAD

**Discussion by mengfangchao** [kubernetes-client/csharp#1295] [Q&A]
[IBasicKubernetes] interfice file doesnot found; so i cannot use the method like [CustomObjects.GetClusterCustomObjecAsync]

**Comment by tg123** [comment]
i do not get your idea

here is GetClusterCustomObjecAsync
https://kubernetes-client.github.io/csharp/api/k8s.CustomObjectsOperationsExtensions.html#k8s_CustomObjectsOperationsExtensions_GetClusterCustomObjectAsync_k8s_ICustomObjectsOperations_System_String_System_String_System_String_System_String_System_Threading_CancellationToken_
